### PR TITLE
More endpoint language edits (do not merge)

### PR DESCRIPTION
**Pull request by j-rigg**
Hi @kirillta and @mingazhev,

Here's my updated edits for the endpoint descriptions.

Cheers,

Jackson

### CODE REVIEW

File: HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml [hunk 1]
```diff
@@ -432,7 +432,7 @@
         </member>
         <member name="P:HappyTravel.Edo.DirectApi.Models.Search.RoomContract.IsAvailableImmediately">
             <summary>
-                Flag indicates if the contract can be booked immediately. 'FALSE' means what the contract is available on request.
+                Indicates if you can book the contract immediately (`False` means that the contract is available by special request)
```
Comment (j-rigg) [inline HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml:435]: If this is `false`, what does the client have to do? Do they need to contact the accommodation?

Comment (kirillta) [reply HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml:435]: No, not really, that means what accommodation staff need to check availability manually, i.e. they have no CRM system. That means we can't provide an instant confirmation for a booking, and a client have to wait for several hours.

File: HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml [hunk 2]
```diff
@@ -432,7 +432,7 @@
         </member>
         <member name="P:HappyTravel.Edo.DirectApi.Models.Search.RoomContract.IsAvailableImmediately">
             <summary>
-                Flag indicates if the contract can be booked immediately. 'FALSE' means what the contract is available on request.
+                Indicates if the accommodation can immediately confirm the booking (If `false`, the accommodation must manually check availability, which can take several hours)
```
Comment (j-rigg) [inline HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml:435]: Corrected this line

File: HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml (hunk 1 above)
Comment (j-rigg) [reply HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml]: Gotcha! I updated this. See the comment below.

**Comment by kirillta** [comment]
Sorry, we are late with new sections

File: HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml (hunk 1 above)
Comment (kirillta) [reply HappyTravel.Edo.DirectApi/HappyTravel.Edo.DirectApi.xml]: Added in #1464

**Review by kirillta — APPROVED** [review]

**Comment by j-rigg** [comment]
Hi @kirillta, 
Me too.
Sure, I'm already done the booking status part. Shouldn't take long.
I noticed that the spelling is "Happy Travel" on your website but "HappyTravel" in the API overview (without a space). Should I update that? I think the only change you'd have to make manually is the title.

**Comment by kirillta** [comment]
> "Happy Travel"

That's quite common name, so we try to stick with "Happytravel.com" for all API-related texts

**Comment by j-rigg** [comment]
> That's quite common name, so we try to stick with "Happytravel.com" for all API-related texts

Cool - I changed this in a few places for consistency.

**Pull request closed without merging**